domains/measurements/Measurements.API/API/Models/Measurement.cs

```diff
@@ -1,6 +1,5 @@
 namespace API.Models;
 
-//NOTE: This DTO is a reflection of the output from the DataSync API.
```

Comment:
Why delete this?

Reply:
I added it back: https://github.com/Energinet-DataHub/energy-origin/pull/539/commits/a4abb707214598c4c2adb390115852b1fd970f81